platform/android/java/lib/src/org/godotengine/godot/input/GodotGestureHandler.java

```diff
@@ -70,15 +72,17 @@ public void onLongPress(MotionEvent event) {
 		//Log.i("GodotGesture", "onLongPress");
 	}
 
+	@RequiresApi(api = Build.VERSION_CODES.M)
```

Comment:
This typically doesn't work well for Android software that needs to support older versions like Godot. If a piece logic is gated by a version, when you should use a `if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.M)` check in order to use the logic.